src/header/name.rs

```diff
@@ -990,7 +994,7 @@ const HEADER_CHARS: [u8; 256] = [
         0,     0,     0,     0,     0,  b'a',  b'b',  b'c',  b'd',  b'e', //  6x
      b'f',  b'g',  b'h',  b'i',  b'j',  b'k',  b'l',  b'm',  b'n',  b'o', //  7x
      b'p',  b'q',  b'r',  b's',  b't',  b'u',  b'v',  b'w',  b'x',  b'y', //  8x
-     b'z',     0,     0,     0,     0,  b'_',     0,  b'a',  b'b',  b'c', //  9x
```

Comment:
It looks like this removes the `z`?

Reply:
Oh woups... I copy/pasted it from the HTTP2 tokens